### PR TITLE
Mks 22 create api project

### PR DESCRIPTION
**Pull request by tann96**
MKS 22

### CODE REVIEW

**Review by TsisinskyjV — APPROVED** [review]

**Review by Magfiozy — COMMENTED** [review]
Why you committed a lot of unusable files?

File: Tetiana_Sulyma/Tetiana_Test/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+
+            services.AddSwaggerGen(c =>
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Startup.cs]: It is another Task.

File: Tetiana_Sulyma/Tetiana_Test/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            app.UseStaticFiles();
+            app.UseSwagger();
+            app.UseSwaggerUI(c =>
+            {
+                c.SwaggerEndpoint("swagger/v1/swagger.json", "My API V1");
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Startup.cs]: The same, another task.

File: Tetiana_Sulyma/Tetiana_Test/Tetiana_Test.csproj [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1" />
+    <PackageReference Include="Swashbuckle.AspNetCore.Swagger" Version="4.0.1" />
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Tetiana_Test.csproj]: Same

File: Tetiana_Sulyma/Tetiana_Test/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "iisExpress": {
+      "applicationUrl": "http://localhost:58382",
+      "sslPort": 44359
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
```
Comment (Magfiozy) [inline Tetiana_Sulyma/Tetiana_Test/Properties/launchSettings.json]: What is it?

File: Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs [hunk 5]
```diff
@@ -0,0 +1,44 @@
+﻿using System.Linq;
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs]: Is it unused using?

File: Tetiana_Sulyma/Tetiana_Test/Startup.cs (hunk 1 above)
Comment (tann96) [reply Tetiana_Sulyma/Tetiana_Test/Startup.cs]: Done

File: Tetiana_Sulyma/Tetiana_Test/Startup.cs (hunk 2 above)
Comment (tann96) [reply Tetiana_Sulyma/Tetiana_Test/Startup.cs]: Done

File: Tetiana_Sulyma/Tetiana_Test/Tetiana_Test.csproj (hunk 3 above)
Comment (tann96) [reply Tetiana_Sulyma/Tetiana_Test/Tetiana_Test.csproj]: Done

File: Tetiana_Sulyma/Tetiana_Test/Properties/launchSettings.json (hunk 4 above)
Comment (tann96) [reply Tetiana_Sulyma/Tetiana_Test/Properties/launchSettings.json]: Done

File: Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs (hunk 5 above)
Comment (tann96) [reply Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs]: Done

**Review by bvzayats — APPROVED** [review]

**Pull request merged**